src/metabase/metabot.clj

```diff
@@ -2,12 +2,13 @@
   "The core metabot namespace. Consists primarily of functions named infer-X,
   where X is the thing we want to extract from the bot response."
   (:require
-   [cheshire.core :as json]
-   [metabase.lib.native :as lib-native]
-   [metabase.metabot.client :as metabot-client]
-   [metabase.metabot.settings :as metabot-settings]
-   [metabase.metabot.util :as metabot-util]
-   [metabase.util.log :as log]))
+    [cheshire.core :as json]
```

Comment:
this indentation change seems weird. not sure which is right offhand but i don't like that the level changed. is this related to your config loss?

Reply:
From other namespaces, it should be underneath the `:`. (and getting this wrong creates painful merge conflicts that have no substance to them, just an extra space)

Reply:
I don't believe the indentation is fixed. The diff here still shows all of the rows indented a single space. And comparing to other namespaces it is more indented

```clojure
(ns metabase.api.card-test
  "Tests for /api/card endpoints."
  (:require
   [cheshire.core :as json]
   [clojure.set :as set]
```

Reply:
Good catch. I thought I'd looked at every file.